firefox_puppeteer/ui/windows.py

```diff
@@ -293,14 +300,15 @@ def get_property(self, property_id):
         """
         return self._l10n.get_property(self.properties, property_id)
 
-    def open_window(self, callback=None, expected_window_class=None):
+    def open_window(self, callback=None, expected_window_class=None, is_focused=True):
         """Opens a new top-level chrome window.
 
         :param callback: Optional, function to trigger the window to open. It is
          triggered with the current :class:`BaseWindow` as parameter.
          Defaults to `window.open()`.
-
         :param expected_class: Optional, check for the correct window class.
+        :param is_focused: Optional, waits until the new window will be focused.
+         Defaults to `True`.
```

Comment:
I don't think you want to do anything with a window in case it hasn't the focus yet, which is the case in most of the tests. So this simply eases test creation by not having to call focus() or have a wait() call each time a new window is opened.